### PR TITLE
Implement java.lang.StrictMath.IEEEremainder() native method.

### PR DESCRIPTION
**Pull request by CoderRooftrellen**
…test case

### CODE REVIEW

File: src/natives/java_lang.ts [hunk 1]
```diff
@@ -888,10 +888,45 @@ class java_lang_StrictMath {
...
+      return 0 * x; // Get correct sign.
+
+    // Achieve x < 2y, then take first shot at remainder.
+    if (y < TWO_1023)
+      x %= y + y;
+
+    // Now adjust x to get correct precision.
+    if (y < 4 / TWO_1023)
+      {
+        if (x + x > y)
```
Comment (perimosocordiae) [inline src/natives/java_lang.ts]: This is very nitpicky, but this project uses curly braces on the same line as the if/for statement.

**Comment by CoderRooftrellen** [comment]
Format updated.

**Comment by CoderRooftrellen** [comment]
@perimosocordiae 
Hi, It looks like that it has a CI error.
I have run the test case locally and it looks the test case has passed.
do you know what's caused this error?

**Comment by jvilk** [comment]
@CoderRooftrellen Yes, there is a build error. I have a fix locally that I will push momentarily.

**Comment by CoderRooftrellen** [comment]
@jvilk 
is there anything I need to do for the build error ?

**Comment by jvilk** [comment]
@CoderRooftrellen Nope, I'm working on fixing the build error right now, then I'll test your changes. I'll let you know if there's anything you need to do.

Normally, I'd merge this in faster, but I'm at a conference until Saturday. :) Hopefully I'll be able to right the build tonight and merge this in.

**Pull request merged**